test/helpers/AdapterTest.php

```diff
@@ -339,12 +339,14 @@ public function test_tables()
 
 	public function test_query_column_info()
 	{
-		$this->assert_greater_than(0,count($this->conn->query_column_info("authors")));
+		$query_info = $this->conn->query_column_info("authors");
+		$this->assert_true($query_info instanceof PDOStatement);
```

Comment:
@koenpunt No, you can't do that actually. The PDOStatement is callable but not countable. In PHP 7.2 this will trigger this warning and errors in PHPUnit:

> 1) MysqlAdapterTest::test_query_column_info
count(): Parameter must be an array or an object that implements Countable

An instanceof check here ensures you know you ran a valid query that actually produced a PDOStatement. I'd recommend leaving this as it is.